src/lt/objs/files.cljs

```diff
@@ -231,13 +233,13 @@
       (when cb (cb e)))))
 
 (defn trash! [path]
-  (.moveItemTotrash shell path))
+  (.moveItemTotrash electron-shell path))
 
 (defn delete!
   "Delete file or directory"
   [path]
   (if (dir? path)
-    (.rmdirSyncRecursive wrench path)
+    (.rm shell "-rf" path)
```

Comment:
Nice catch! You are correct, it does not appear force is needed based on some quick testing I just did. I have opened PR #2284 to address this.